docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -1,277 +1,105 @@
-provides a free Upgrade Assistant that identifies issues you need to address
-before upgrading and simplifies migrating indices that need to be reindexed
-before you upgrade. The Upgrade Assistant is enabled with both Trial and
-Basic licenses. You can install {xpack} solely for the purpose of upgrading.
-
-Rolling upgrades are supported when upgrading from Elasticsearch 5.6 and
-Elasticsearch 6.0-6.2 to {version}. Upgrading from any
-version prior to 5.6 requires a full cluster restart.
-
-IMPORTANT: 2.x indices are not compatible with {version}. You must
-remove or reindex them on your 5.n cluster before upgrading to {version}. The internal
-Kibana and {xpack} indices and the default Beats and Logstash mapping templates
-also need to be updated to work with {version}.
+IMPORTANT: 5.x indices are not compatible with {version}. You must
+remove or reindex them to upgrade to {version}. The default Beats and
+Logstash mapping templates also need to be updated to work with {version}.
 
 === Preparing to upgrade
 
 Before upgrading the Elastic Stack to {version}:
 
-. Back up your data. You **cannot roll back** to an earlier version unless
-you have a backup of your data. For information about creating snapshots, see
-{ref}/modules-snapshots.html[Snapshot and Restore].
-
-. Check the Elasticsearch {ref}/settings.html[deprecation log] to see if
-you're using any deprecated features and update your code accordingly.
-By default, deprecation log messages are enabled at the `WARN` level.
-
-. Review the breaking changes for each product you use
-and make the necessary changes so your code is compatible with {version}. See
-<<elastic-stack-breaking-changes>>.
-
-. {ref}/docs-reindex.html[Reindex] or delete any indices created on 2.n. We recommend
-upgrading to the most recent 5.6 and using the {xpack} Reindex Helper to reindex 2.n indices.
+. Check the {es} {ref}/logging.html#deprecation-logging[deprecation log]
+to see if you're using any deprecated features and update your code accordingly.
+By default, deprecation warnings are logged when the log level is set to `WARN`.
 
-. If Kibana and {xpack} are part of your stack, upgrade the internal Kibana
-and {xpack} indices. We recommend using the {xpack} 5.6 Reindex Helper to
-upgrade the internal indices. If you're performing a full cluster restart upgrade
-from an earlier version, you can also
-<<upgrade-internal-indices,use the `_migration/upgrade` API>> directly to
-upgrade the internal indices after you install Elasticsearch {version}.
+. Review the <<elastic-stack-breaking-changes>> and upgrade your code to work
+with {version}.
 
-. If you use {stack} {security-features} to secure your cluster:
-.. Make sure TLS is enabled to encrypt communications between nodes. TLS must
-be enabled to upgrade to {version}. For more information, see
-{stack-ov}/encrypting-communications.html[Encrypting communications].
+. Upgrade to 6.7 and use the {kib} Upgrade Assistant to {ref}/docs-reindex.html[reindex]
```

Comment:
Since reindex is a link to the API maybe we should add a link to the UI info too?

```suggestion
. Upgrade to 6.7 and use the {kibana-ref}/upgrade-assistant.html[{kib} Upgrade Assistant] to {ref}/docs-reindex.html[reindex]
```